src/user/__snapshots__/user.spec.js.snap

```diff
@@ -0,0 +1,24 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`user .create() create a user successfully creates a user with Permission 1`] = `
```

Comment:
> user .create() create a user successfully creates a user with Permission

Be careful with how `describe`/`it` are nested and their titles, this should result in correct sentences.